packages/gtk4.rb

```diff
@@ -70,9 +69,9 @@ def self.postinstall
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
     # update mime database
     system "update-mime-database #{CREW_PREFIX}/share/mime"
-    if !File.exists?("#{HOME}/.config/gtk-4.0/settings.ini")
+    unless File.exist?("#{HOME}/.config/gtk-4.0/settings.ini")
       puts
-      puts "Adding basic gtk4 settings to ~/.config/gtk-4.0/settings.ini".lightblue
+      puts 'Adding basic gtk4 settings to ~/.config/gtk-4.0/settings.ini'.lightblue
       FileUtils.mkdir_p "#{HOME}/.config/gtk-4.0"
       system "cat << 'EOF' > #{HOME}/.config/gtk-4.0/settings.ini
```

Comment:
Would it make sense to add this in the `def self.build` section and include the config in the package?  The drawback would be the config goes away when the package is removed.

Reply:
The problem is that one might have local customizations to these files.  You need to make sure not to overwrite the settings for these.